internal/presentation/presentation.go

```diff
@@ -590,8 +593,9 @@ func generateResultKeys(rs rego.ResultSet) []resultKey {
 			})
 		}
 
+
 		for i, expr := range rs[0].Expressions {
-			if _, ok := expr.Value.(bool); !ok {
+			if _, ok := expr.Value.(bool); !ok || boolRenderable(rs) {
```

Comment:
I think we can make this check simpler: `len(rs[0].Bindings) == 0`.

This is a bit confusing, but it's correct because:

1. All of the result set (`rs`) elements have the same number of bindings so we do not have to check every single binding list for emptyness--checking the first is adequate.

2. If all of the expression values are false the `prettyResult` function will output `false`.

Reply:
Thank you for your explanations!
This works as you say, so I removed unused functions.